packages/bot-web-ui/src/app/app-main.tsx

```diff
@@ -12,7 +12,26 @@ type TAppProps = {
     };
 };
 
+const originToDomainMap = {
+    'staging-app.deriv.com': 'staging-dbot.deriv.com',
+    'staging-app.deriv.me': 'staging-dbot.deriv.com',
+    'staging-app.deriv.be': 'staging-dbot.deriv.com',
+    'app.deriv.com': 'dbot.deriv.com',
+    'app.deriv.me': 'dbot.deriv.me',
+    'app.deriv.be': 'dbot.deriv.be',
+};
+
 const App = ({ passthrough }: TAppProps) => {
+    // Extract the correct domain based on the current origin
+    const targetDomain = Object.entries(originToDomainMap).find(([origin]) =>
+        window.location.origin.includes(origin)
+    )?.[1];
+
```

Comment:
const targetDomain = originToDomainMap[window.location.host] 

@shafin-deriv can we do this ?

Reply:
Sure can